src/main/java/org/example/Hangman/Hangman.java

```diff
@@ -0,0 +1,94 @@
+package org.example.Hangman;
+
+import java.util.HashSet;
+import java.util.Scanner;
+import java.util.Set;
+
+public class Hangman {
+    public void start() {
+        Player player1 = new Player();
+        Player player2 = new Player();
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("select first players name");
+        player1.setName(scanner.next());
+        player1.setScore(0);
+
+        System.out.println("select second players name");
+        player2.setName(scanner.next());
+        player2.setScore(0);
+
+        boolean endGame = false;
+
+        while(!endGame){
+            printScore(player1,player2);
+            runRuond(player1,scanner);
+            printScore(player1,player2);
+            runRuond(player2,scanner);
+
+            if(player1.getScore()>=3 || player2.getScore()>=3){
+                endGame = true;
+                if(player1.getScore()==player2.getScore()){
+                    System.out.println("draw");
+                } else if (player1.getScore()==3) {
```

Comment:
Skoro wyżej jest warunek >= 3 to tutaj chyba powinien być taki sam? 😛

Reply:
zrobione, mam już na to commita.